### ISSUE TITLE
Feature wanted: highlight span -> select tag -> switch layer -> select tag

### ISSUE THREAD

**Issue by zhh210** [webanno/webanno#1107]
This is related to #1097 about switching layers. I have two layers (L1, L2) each with span tags (L1_t1, L1_t2 for L1 and L2_t1, L2_t2 for L2). What I'm expecting is that: in layer L1, after selecting a text span and choosing a tag from (L1_t1, L1_t2), I'd like to switch layer to L2 from the dropdown on the top-right and keep choosing among L2_t1, L2_t2 for the same selected span. In other words, I'd like to assign tags to the same selected span in different layers. 

If I don't check the "Remember layer" option in the setting, I got the msg:

> You are about to replace an annotation of type L1 with a new annotation of type L2. This action will discard all feature values and delete 0 relation(s)/link(s) attached to current L1 annotation. Are you sure?
what I annotated in L1 will be missing if I confirm the msg.

If I check the "Remember layer" option in the setting, in layer L2, I can only pick among L1_t1, L1_t2 rather than L2_t1, L2_t2. But I really need the annotation in layer L1 be kept while additionally allow me to annotating in L2 with tags L2_t1 and L2_t2.

I'm running with Webanno 3.4.6 standalone version.

**Comment by reckart** [comment]
Ok, so let me describe in my words how I understand what you wish:

* you create an annotation on L1 and assign some tag L1_t1 or L2_t2 to it
* then you wish to create a second annotation at the same position as L1 but on layer L2 and assign some tag L2_t1 or L2_t2 to it

Is that correct?

**Comment by zhh210** [comment]
> Ok, so let me describe in my words how I understand what you wish:
> 
> * you create an annotation on L1 and assign some tag L1_t1 or L2_t2 to it
> * then you wish to create a second annotation at the same position as L1 but on layer L2 and assign some tag L2_t1 or L2_t2 to it
> 
> Is that correct?

**Comment by reckart** [comment]
> Exactly! Currently it seems the only way to do it is to select the span twice and do the annotation separately.

Exactly.

The right sidebar (annotation detail editor panel aka ADEP) is meant to edit annotations, not to create them. Annotations are created in the center part of the window by marking text.

**Remember layer disabled (default)** - in this mode, the changing the layer affects the currently selected annotation. If you want to create an annotation on another layer, you first have to press **cancel** so that no annotation is selected, then select the new layer, and then mark the text you want to annotate. Personally, I find this kind of annoying.

**Remember layer enabled** - in this mode, the layer dropdown works independently from the currently selected annotation. If you want to create an annotation on another layer, you just select the layer from the dropdown and mark the text. No need to press cancel. Personally, I find this much more natural.

**Comment by zhh210** [comment]
My annotation task is to annotate the aspect/topic for layer L1 and annotate a sentiment for the same span in layer L2. As such, each span in a doc will have to be highlighted twice for in each layer separately. It would be really neat if there's a feature to keep the annotated span in current layer saved while allow creating annotation in the new layer.

**Comment by reckart** [comment]
@zhh210 have you considered having a single layer with two features?

**Comment by zhh210** [comment]
@reckart yes but in my usecase of sentiment the number of aspect/topic is large, which multiplied by 5 (very_posotive/very_negative/positive/negative/neutral) is kind of cumbersome for the annotators to choose.

**Comment by reckart** [comment]
Idea: right-click on existing annotation to open a popup with an option "New annotation" which would create at the location of the clicked annotation another annotation of the currently selected layer type.
